docs/providers/documentation/amazonsqs-provider.mdx

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
can you add some example for input/output payload? for example - how event payload should look like for Keep to process it? what are mandatory fields?